<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editar</class>
 <widget class="QDialog" name="Editar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblDescripcion">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
